tests/integration/test_charm.py

```diff
@@ -315,3 +316,17 @@ async def assert_links_in_configmap_by_text_value(
         assert item.text in links_texts
 
     return links_texts
+
+
+@pytest.mark.asyncio
+async def test_dashboard_access(ops_test: OpsTest, lightkube_client: Client):
+    namespace = ops_test.model_name
+    application_ip = lightkube_client.get(Service, CHARM_NAME, namespace=namespace).spec.clusterIP
+    config = await ops_test.model.applications[CHARM_NAME].get_config()
+    application_port = config["port"]["value"]
```

Comment:
```suggestion
    application_port = (await ops_test.model.applications[CHARM_NAME].get_config())["port"]["value"]
```
you could do it as a one-liner since we are not reusing `config`

Reply:
Changed it to a one-liner with the proper form suggested by the linter.